packages/test-helpers/src/asserts/assertThrow.js

```diff
@@ -0,0 +1,110 @@
+const { assert } = require('chai')
+const { decodeErrorReasonFromTx, isGeth } = require('../')
+
+const THROW_ERROR_PREFIX =
+  'Returned error: VM Exception while processing transaction: revert'
+
+async function assertThrows(
+  blockOrPromise,
+  expectedErrorCode,
+  expectedReason,
+  ctx
+) {
+  try {
+    typeof blockOrPromise === 'function'
+      ? await blockOrPromise()
+      : await blockOrPromise
+  } catch (error) {
+    if (isGeth(ctx)) {
```

Comment:
This part is a bit sketchy and taken from [dao-templates](https://github.com/aragon/dao-templates/blob/master/shared/helpers/assertRevert.js).

I think this should still work on an Istanbul context, but I won't know until we try :smile:.

In any case, my philosophy here is to ensure we can use the same utilities regardless of ganache or a real chain and we can certainly come back to this to fix it if it doesn't end up working well.